src/crh/mod.rs

```diff
@@ -23,3 +25,20 @@ pub trait FixedLengthCRH {
     fn setup<R: Rng>(r: &mut R) -> Result<Self::Parameters, Error>;
     fn evaluate(parameters: &Self::Parameters, input: &[u8]) -> Result<Self::Output, Error>;
 }
+
+pub trait TwoToOneCRH {
+    /// The size for left input and right input
+    const LEFT_INPUT_SIZE_BITS: usize;
+    const RIGHT_INPUT_SIZE_BITS: usize;
```

Comment:
```suggestion
    /// The bit size of the left input.
    const LEFT_INPUT_SIZE_BITS: usize;
    /// The bit size of the right input.
    const RIGHT_INPUT_SIZE_BITS: usize;
```

Reply:
Also, is there a case where these two are unequal?

Reply:
I can't think of any...

Reply:
@ValarDragon ?